test/integration/mapping_validation_test.rb

```diff
@@ -111,6 +111,37 @@ def setup
     end
   end
 
+  test "Shopify taxonomy version is in consistent between VERSION file and mappings in the /data folder" do
+    shopify_taxonomy_version_from_file = "shopify/" + @sys.read_file("VERSION").strip
+    mapping_rule_files = @sys.glob("data/integrations/*/*/mappings/*_shopify.yml")
+    files_include_inconsistent_shopify_taxonomy_version = []
+    mapping_rule_files.each do |file|
+      raw_mappings = @sys.parse_yaml(file)
+      [raw_mappings["input_taxonomy"], raw_mappings["output_taxonomy"]].each do |taxonomy_version|
+        next if !taxonomy_version.include?("shopify") || taxonomy_version.include?("shopify/2022-02")
```

Comment:
Similar to above comment, ideally we shouldn't be doing YAML file loading/parsing here to check the taxonomy version and instead should be building these kinds of checks against our models

Reply:
> checks against our models 

As we are doing checks in tests and the DB states are empty when accessing models e.g., MappingRule from tests, that's why I load data from YAML files directly.